tardis/plasma/properties/atomic.py

```diff
@@ -108,8 +109,17 @@ def calculate(self, atomic_data, selected_atoms):
 class IonizationData(ProcessingPlasmaProperty):
     name = 'ionization_data'
 
-    def calculate(self, atomic_data, selected_atoms):
+    def calculate(self, atomic_data, levels):
         if self.value is not None:
             return self.value
         else:
             return atomic_data.ionization_data
+
+class ZetaData(ProcessingPlasmaProperty):
+    name = 'zeta_data'
```

Comment:
@wkerzendorf I'd like to make `ZetaData` a `BaseAtomicDataProperty` so that it could be filtered in the same way as the other atomic properties. That would require explicitly naming the indices in the atomic datafiles as `atomic_number` and `ion_number`. Could we change that in current/future datasets? Also, in some datasets, e.g. the `included_he_atomic_data` included in `conftest.py`, some atomic properties (`lines`, `levels`) are named with an underscore in front. What's the reason for that? I've noticed it's true in some datasets but not in others I use. It's just important to know which one is the standard one so the plasma can find it.